app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(player: list) -> int:
+    return sum([player.get_rating() for player in player])
+
+
+def elves_concert(elf: list) -> None:
+    [el.play_elf_song() for el in elf]
```

Comment:
same thing with [] here, it's better to use for loop to not make a list in memory